karaf/features/src/main/feature/feature.xml

```diff
@@ -260,7 +260,7 @@
             https://github.com/square/okhttp/pull/2246
         -->
         <!-- TODO use ServiceMix bundle for okhttp when dependency on okio is fixed there -->
-        <bundle dependency="true">wrap:mvn:com.squareup.okhttp/okhttp/2.2.0$Import-Package=org.apache.http.*;resolution:=optional,android.util.*;resolution:=optional,okio;version=%27[1.2.0,1.3.0)%27,*</bundle>
+        <bundle dependency="true">mvn:org.apache.servicemix.bundles/org.apache.servicemix.bundles.okhttp/2.2.0_2</bundle>
```

Comment:
Can you explain your reasoning for this ... I remember there being an issue with the servicemix wrapped version

Also version should be parameterised